iron-a11y-keys-behavior.html

```diff
@@ -411,14 +374,21 @@
           event.stopPropagation();
         }
 
-        keyBindings.forEach(function(keyBinding) {
-          var keyCombo = keyBinding[0];
-          var handlerName = keyBinding[1];
+        // if event has been already prevented, don't do anything
+        if (event.defaultPrevented) {
+          return;
+        }
 
-          if (!event.defaultPrevented && keyComboMatchesEvent(keyCombo, event)) {
+        var eventKey = normalizedKeyForEvent(event);
+        for (var i = 0, l = keyBindings.length; i < l; i++) {
+          var keyCombo = keyBindings[i][0];
+          var handlerName = keyBindings[i][1];
+          if (keyComboMatchesEvent(keyCombo, event, eventKey)) {
             this._triggerKeyHandler(keyCombo, handlerName, event);
+            // found the matching event, exit from the loop
+            return;
```

Comment:
One case could be when we have `keyBindings: "a shift+a shift+ctrl+a"`
In that case `a` listener would be triggered and the other 2 not. I would solve this by sorting the `keyBindings` by checking first the ones with modifiers, then the ones without modifier.